cart/interfaces/templatefunctions/getItemAdjustment.go

```diff
@@ -0,0 +1,95 @@
+package templatefunctions
+
+import (
+	"context"
+
+	"flamingo.me/flamingo-commerce/v3/cart/application"
+	"flamingo.me/flamingo/v3/framework/web"
+
+	"flamingo.me/flamingo-commerce/v3/cart/domain/cart"
+)
+
+type (
+	// GetQuantityAdjustmentDeletedItemsMessages is exported as a template function
+	GetQuantityAdjustmentDeletedItemsMessages struct{}
+
+	// GetQuantityAdjustmentUpdatedItemsMessage is exported as a template function
+	GetQuantityAdjustmentUpdatedItemsMessage struct{}
+
+	// RemoveQuantityAdjustmentMessages is exported as a template function
+	RemoveQuantityAdjustmentMessages struct{}
+
+	// QuantityAdjustment is returned by the template function
+	QuantityAdjustment struct {
+		Item         cart.Item
+		DeliveryCode string
+		PrevQty      int
+		CurrQty      int
+	}
+)
+
+// Func defines the GetQuantityAdjustmentDeletedItemsMessages template function
+func (gdm *GetQuantityAdjustmentDeletedItemsMessages) Func(ctx context.Context) interface{} {
+	return func() []QuantityAdjustment {
+		session := web.SessionFromContext(ctx)
+
+		deletedAdjustments := make([]QuantityAdjustment, 0)
+
+		if adjustmentsI, found := session.Load("cart.view.quantity.adjustments"); found {
+			if adjustments, ok := adjustmentsI.(application.QtyAdjustmentResults); ok {
```

Comment:
Could you use a more speaking name than adjustmentsI? :D
For me reading it this suggests an interfaces.

Maybe sessionAdjustments?
loadedAdjustments?
foundAdjustments?

Reply:
Renamed to sessionAdjustments